python-lib/ocr_utils.py

```diff
@@ -0,0 +1,102 @@
+import dataiku
+from dataiku.customrecipe import get_input_names_for_role, get_output_names_for_role
+from io import BytesIO
+from ocr_constants import Constants
+import os
+import pypdfium2 as pdfium
+from shutil import which
+
+
+def get_input_output(input_type='dataset', output_type='dataset'):
```

Comment:
[nitpick] The various helper functions relative to recipe config belong more in a `recipe_utils` or `recipe_io_utils`, wdyt ?

Reply:
new `ocr_recipes_io_utils.py` file added in [6bd2646](https://github.com/dataiku/dss-plugin-tesseract-ocr/pull/70/commits/6bd2646f11f98fb829fd0352450b254c56963218)
Starting the file names with a plugin specific name is a good practice for plugins to not have the same module names as in the user python-lib of the project/instance because they are both in the same PATH